compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -433,127 +471,75 @@ class JSExportTest extends DirectTest with TestHelpers {
   }
 
   @Test
-  def noImplicitNameNestedExportClass: Unit = {
```

Comment:
Why is this test removed? I would assume the error messages would change, but we should still make sure that the snippets in this test are rejected. Or are they tested by some other test already?